npm/test/upload-form-files.test.ts

```diff
@@ -0,0 +1,565 @@
+import "@testing-library/jest-dom"
```

Comment:
There appears to be quite a bit of shared test setup stuff between this set of tests and the `upload-form-folder.test.ts` Wondering if its possible to create `upload-form-utils` file and put common stuff in there, and import it into each of the different tests sets? Would reduce the amount of duplicate code, and make the tests themselves more visible. What do you think? Not sure that would even work with Jest tests.